package.json

```diff
@@ -56,8 +57,8 @@
     "lighthouse": "^12.1.0",
     "load-grunt-config": "^3.0.1",
     "lodash-webpack-plugin": "^0.11.6",
+    "postcss-scss": "^4.0.9",
```

Comment:
Why do we need this package?

Reply:
@jkanive We need `"postcss-scss"` since our styles are written in SCSS, which has features like `//` comments and `$variables` that regular CSS doesn't support. Without this parser, Stylelint doesn't understand the syntax and throws errors like
```
assets/scss/checkout.scss
  1:1  ✖  Unknown word //  CssSyntaxError.
```
So it helps Stylelint read our SCSS files properly.
We didn’t need `postcss-scss` before because we were using the `stylelint-config-sass-guidelines` config, which internally handled the SCSS parsing.